gson/src/test/java/com/google/gson/GsonBuilderTest.java

```diff
@@ -254,4 +255,32 @@ public void testSetStrictness() throws IOException {
     assertThat(gson.newJsonReader(new StringReader("{}")).getStrictness()).isEqualTo(STRICTNESS);
     assertThat(gson.newJsonWriter(new StringWriter()).getStrictness()).isEqualTo(STRICTNESS);
   }
+
+  @Test
+  public void testRegisterTypeAdapterForObjectAndJsonElements() {
+    Type[] types = {
+            Object.class,
+            JsonElement.class,
+            JsonArray.class,

```

Comment:
The indentation here looks wrong, I think it should look like this, but I am not completely sure:
```suggestion
        Object.class,
        JsonElement.class,
        JsonArray.class,
```

Reply:
I reformatted the code using code editor (IntelliJ), seems ok, thanks for pointing out.

Reply:
You haven't pushed the changes yet though, right?

Reply:
Not yet